src/pytest_dbfixtures/factories/elasticsearch.py

```diff
@@ -75,7 +75,13 @@ def elasticsearch_proc_fixture(request):
                 index_store_type
             )
         else:
-            index_store_type_option = ''
+            elasticsearch, _ = try_import('elasticsearch', request)
+            if elasticsearch.__version__[0] < 2:
```

Comment:
The reason to use in-memory indexes was a performance gain. To not loose this gain, in case elasticsearch version < 2.0 is used, a condition checks for this case and keeps the original default setting ('memory').

Version info is taken from the python module, as [documentation](https://elasticsearch-py.readthedocs.org/en/master/) demands the usage of matching major versions between elasticsearch server and python client.